projects/wallet-template/src/containers/WalletPopup/pages/Accounts.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  Settings,
+  Plus,
+  ChevronRight,
+  ArrowRightLeft,
+  RotateCcw,
+  PlusCircle,
+} from "lucide-react"
+import { Link, useNavigate } from "react-router-dom"
+import { ButtonHTMLAttributes, useEffect, useState } from "react"
+import { Keyset } from "../../../background/types"
+import { fromHex, toHex } from "@polkadot-api/utils"
+import { entropyToMiniSecret, ss58Address } from "@polkadot-labs/hdkd-helpers"
+import { sr25519CreateDerive } from "@polkadot-labs/hdkd"
+import { rpc } from "../api"
+
+export const Accounts = () => {
+  const navigate = useNavigate()
+  const [keysets, setKeysets] = useState<Record<string, Keyset>>({})
+
+  const keysetsLength = Object.keys(keysets).length
+
+  const reset = async () => {
+    await rpc.client.clearKeysets()
+    navigate(0)
+  }
+
+  useEffect(() => {
+    ;(async () => {
+      const existingKeySets = await rpc.client.listKeysets()
+      setKeysets(existingKeySets)
+    })()
+  }, [])
```

Comment:
use `useSwr` with https://swr.vercel.app/docs/mutation#bound-mutate

Reply:
```
  const { data: keysets } = useSWR("/rpc/keysets", async () => {
    return rpc.client.listKeysets()
  })

```

just this works right?

Reply:
yes

for a naming convention, I would suggest `rpc.listKeysets`

`rpc.methodName`